src/backends/backend/triton_backend_manager.h

```diff
@@ -143,9 +143,14 @@ class TritonBackendManager {
       const BackendCmdlineConfig& backend_cmdline_config,
       std::shared_ptr<TritonBackend>* backend);
 
+  static const std::unordered_map<std::string, std::vector<std::string>>&
+  BackendState();
+
  private:
-  std::mutex mu_;
```

Comment:
Why did you change this mutex name? As a convention we typically use mu_. Can you put it back like it was?